assets/js/modules/analytics/datastore/report.js

```diff
@@ -72,6 +72,13 @@ const fetchGetReportStore = createFetchStore( {
 			);
 		}
 
+		if ( dimensionFilters ) {
+			invariant(
+				isValidDimensionFilters( dimensionFilters, dimensions ),
+				'Dimensions filters must be an object where the keys are valid Dimensions and values are non-zero length strings.',
```

Comment:
We probably need to adjust this message since we expect values not to be just non-zero strings.

Reply:
Addressed.